app/controllers/api/items_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::ItemsController < ApiController
```

Comment:
Nice, using inheritance to inherit those methods from that class. You could also use a concern/module to mix it in...